src/proto/flwr/proto/exec.proto

```diff
@@ -52,3 +57,9 @@ message ListRunsResponse {
   map<uint64, Run> run_dict = 1;
   string now = 2;
 }
+
+message LoginRequest {}
+message LoginResponse { map<string, string> login_details = 1; }
```

Comment:
Could we list out the login details needed, instead of using a dictionary? Just to make it clear.